resources/i18n/de/strings.xml

```diff
@@ -81,9 +81,9 @@
   <!-- XTXT: Admission State - OTHER Badge Text -->
   <string name="ADMISSION_STATE_OTHER_BADGE_TEXT">""</string>
   <!-- XTXT: Admission State - OTHER Card Body -->
-  <string name="ADMISSION_STATE_OTHER_LONG_TEXT">"Ihre Zertifikate erfüllen derzeit keine Regel."</string>
+  <string name="ADMISSION_STATE_OTHER_LONG_TEXT">"Der G-Status ist zur Zeit nicht relevant und wird daher nicht von der App ausgewiesen."</string>
```

Comment:
How about: "Zertifikate werden zur Zeit nicht geprüft und Ihr Status daher nicht von der App ausgewiesen."

Reply:
@KathrinLuecke 

> "Zertifikate werden zur Zeit nicht geprüft und Ihr Status daher nicht von der App ausgewiesen."

Wouldn't that statement be misleading if certificates are being checked to find out whether mask wearing is mandatory?